### ISSUE TITLE
Allow arosien to commit to repo

### ISSUE THREAD

**Issue by arosien** [apache/daffodil-vscode#88]
Do I need to ask for this on the mailing list? I forget.

**Comment by Shanedell** [comment]
@arosien We don't commit directly to the repository. Usually the process is

- Fork this repository to your GitHub account
- Push branches for code updates to your fork.
- Once you believe your branch is ready to be merged into the Apache repo, then you need to create a pull request from your fork's branch into the Apache's repository `main` branch.

However, if you are referring to merging branches into the `apache/daffodil-vscode` `main` branch, that could be better explained by either @stevedlawrence or @mbeckerle than myself.

**Comment by mbeckerle** [comment]
I will take this up on separate email thread. Closing as this isn't really a ticket.
